resources/views/index.blade.php

```diff
@@ -31,8 +31,13 @@
                 <div class="word font">
                     <p>お気に入りに追加</p>
                 </div>
-                <a href=""><img src="./css/rogo_okiniiri.jpg" alt="お気に入り星" class="rogo"></a>
-                <a href=""><img src="./css/rogo_addreview.jpg" alt="口コミ鉛筆" class="rogo"></a>
+                <form action="{{ route('favorites.store') }}" method="POST">
+                    @csrf
+                    <!-- お気に入りを押してstoreとuserのidをDBに送る -->
+                        <input type="hidden" name="store_id" value="<?php echo $detail['store_id']; ?>">             
```

Comment:
急にphpの書き方がでちゃってますw
ここはこんな感じですね！

```suggestion
                        <input type="hidden" name="store_id" value="{{ $detail->id }}">             
```

Reply:
せっかく中川さんがいい手本を並べてくださっているのに全無視してました…。
店の情報が入ったセクションでstoresテーブルの中のidカラムのデータを送るという認識であっていますか？
この場合、FavoriteContorollerでも`$favorite -> store_id = $request ->  id;`に変更するということですよね？

Reply:
ちがいますー！
この `$detail` はstoreテーブルの中身がはいっているので、お店のidがほしい時は `$detail->id` ですが
Controllerのメソッドの中では$requestで送られてきた値が入るので、そこにはこのviewで書かれているinputタグのname属性の値で入ります！
つまり `name="store_id"` なので 
コントローラーでは $favorite -> store_id = $request -> store_id;
ですね！！

Reply:
FavoriteContorollerのメソッドの中で
`dd($request);` と一度してみてどんな値が送られてきているかデバッグで確認しながらやってみるといいですよ！:+1:

Reply:
コントローラーでstoreテーブルのidを送るときにnameをstore_idにしているからstore_idであっていますね！
一瞬valueの方かと勘違いして慌てました。復習して大丈夫になりました。
ありがとうございます。ddを省きがちでした…。